cirq/contrib/acquaintance/permutation.py

```diff
@@ -89,7 +90,8 @@ def _circuit_diagram_info_(self, args: protocols.CircuitDiagramInfoArgs
 class SwapPermutationGate(PermutationGate):
     """Generic swap gate."""
 
-    def __init__(self, swap_gate: ops.Gate=ops.SWAP):
+    def __init__(self, swap_gate: Optional[ops.Gate] = None):
+        swap_gate = ops.SWAP if swap_gate is None else swap_gate
```

Comment:
This functional change is not related to the PR.

Reply:
This was a hack to avoid a mypy complaint that doesn't seem to be happening anymore.